pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -96,17 +96,17 @@ object CalmTransformer
           .exists(_.startsWith("AMSG"))
     }
 
-  def transmissionIsNo(record: CalmRecord): Boolean =
+  // We suppress unless CatalogueStatus exists and is valid
+  def catalogueStatusIsSuppressible(record: CalmRecord): Boolean =
     record
-      .get("Transmission")
-      .exists { value =>
-        value.toLowerCase match {
-          case "no"  => true
-          case "yes" => false
-          case _ =>
-            info(
-              s"Unrecognised value for Transmission field; assuming 'Yes': $value")
+      .get("CatalogueStatus")
+      .forall { value =>
+        value.toLowerCase.trim match {
+          // As much as it might not look like it, these values mean it should
+          // not be suppressed.
+          case "catalogued" | "not yet available" | "partially catalogued" =>
```

Comment:
**BIKESHED / NANO-NIT** I think my preference would be to have each string as a separate case

Reply:
Any idea why? I find this more readable?

Reply:
I think I prefer this as this is a list of valid `CatalogueStatus`es, and that is what I am trying to reflect.

What about having a `val validCatalogueStatuses = List("catalogued", "not yet available", "partially catalogued")`  

and then `case status if validStatuses.contains(status) => true`

Reply:
What I am trying to express is that these three are _the_ valid options that make this return true.

On a new line makes it feel like the terms aren't connected, which they are.

Reply:
Saying that - what about

```scala
List("catalogued", "not yet available", "partially catalogued").contains(str) match {
  case true => false
  case false => true
}
```

Reply:
```scala
def catalogueStatusSuppressesRecord(record: CalmRecord): Boolean =
    record
      .get("CatalogueStatus")
      .forall(isValidCatalogueStatus)

  def isValidCatalogueStatus(str: String) =
    List("catalogued", "not yet available", "partially catalogued").contains(
      str)
```

Reply:
```scala
val valid = List("catalogued", "not yet available", "partially catalogued").contains(str)
(true, false) match {
  case (t, _) if t == valid => false
  case (_, f) if f == valid => true
}
```

Reply:
My vote is for the original, I don't reckon it's getting any clearer

Reply:
Not even the last attempt?

I've looked through the code, and there's no similar example [bar here](https://github.com/wellcomecollection/catalogue/blob/master/pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraConcepts.scala#L92), and even that isn't really the same.

I'm going to go with the original and leave it to the next dev who touches this to then decide.